src/utils/getUserById.tsx

```diff
@@ -0,0 +1,5 @@
+import usersFromServer from '../api/users';
+
+export function getUserById(todoId: number) {
+  return usersFromServer.find(user => todoId === user.id) || null;
```

Comment:
[CODE STYLE] - The parameter inside the find callback should be named to reflect that it represents a user, for example 'user => userId === user.id'.